tests/FSharp.Compiler.Service.Tests/xunit.runner.json

```diff
@@ -1,5 +1,4 @@
 ﻿{
-    "$schema": "https://xunit.net/schema/current/xunit.runner.schema.json",
-    "appDomain": "ifAvailable",
-    "shadowCopy": false
+  "$schema": "https://xunit.net/schema/current/xunit.runner.schema.json",
+  "appDomain": "denied"
```

Comment:
Are there any negatives for NET472 test execution ?
I could imagine them for isolation in case of crashes etc. (StackOverfowException and friends..)

Reply:
I haven't noticed any worse behavior than before but that's something I'd have to look into. For net472 we do create app domains on our own, to load and execute compiled code: https://github.com/dotnet/fsharp/blob/f44e2b2583799902234236e3e8298e9c4e20c5b5/tests/FSharp.Test.Utilities/CompilerAssert.fs#L361-L362

This change just removes the top level appdomain xUnit would create for the whole assembly. In practice the test execution should be no different than for net9.0 now. It means a test case can crash the whole testhost easily, for example by throwing from a background thread, but we should get a diagnostic telling which one crashed (if we're lucky).

Reply:
Good, thanks for the explanation.